src/Razor/test/Microsoft.NET.Sdk.Razor.Test/IntegrationTests/StaticWebAssetsIntegrationTest.cs

```diff
@@ -271,57 +245,4 @@ private string GetExpectedManifest()
 </StaticWebAssets>";
         }
     }
-
-    public class PackageTestProjectsFixture
```

Comment:
What's the explanation for why this isn't needed anymore?

Reply:
I'm trying to get it done at build time. It was causing some flakyness on the build